elvis.config

```diff
@@ -8,7 +8,8 @@
         rules => [{elvis_style, god_modules, #{ignore => [egithub]}},
                   {elvis_style, dont_repeat_yourself, #{min_complexity => 20}},
                   {elvis_style, line_length, #{limit => 120}},
-                  {elvis_style, state_record_and_type, disable}],
+                  {elvis_style, state_record_and_type, disable},
+                  {elvis_style, invalid_dynamic_call, #{ignore => [oc_chain_reporter]}}],
```

Comment:
it makes sense to exclude this but it'd be nice to have a short comment on why for later readers.